code/modules/mob/living/carbon/human/species.dm

```diff
@@ -50,6 +50,8 @@
 	var/siemens_coeff = 1 //base electrocution coefficient
 	var/damage_overlay_type = "human" //what kind of damage overlays (if any) appear on our species when wounded?
 	var/fixed_mut_color = "" //to use MUTCOLOR with a fixed color that's independent of dna.feature["mcolor"]
+	var/list/species_clothing_blacklist //If you're in this list, you can't wear it
```

Comment:
these don't go on the species, they go on the clothing (like I told you too, and like you actually did (??))